testcafe/tests/pages/drive-model-public.js

```diff
@@ -62,27 +86,50 @@ export default class PublicDrivePage {
       .expect(this.btnPublicMobileDownload.exists)
       .notOk('Create my Cozy Button (mobile) exists')
     await isExistingAndVisibile(this.logo, 'Logo')
-    await isExistingAndVisibile(this.toolbarFilesPublic, 'toolbarFiles')
+    if (type == 'file') {
+      await isExistingAndVisibile(this.toolbarFilesPublic, 'toolbarFiles')
+      await isExistingAndVisibile(
+        this.btnPublicCreateCozyFile,
+        'Create my Cozy Button'
+      )
+    } else if (type == 'folder') {
+      await isExistingAndVisibile(this.toolbarFolderPublic, 'toolbarFolder')
+      await isExistingAndVisibile(
+        this.btnPublicCreateCozyFolder,
+        'Create my Cozy Button'
+      )
+    }
```

Comment:
```js
const isFile = type === 'file' ? true : false
await isExistingAndVisibile(isFile ? this.toolbarFilesPublic : toolbarFolderPublic , 'toolbarFiles')
      await isExistingAndVisibile(
        isFile ? this.btnPublicCreateCozyFile : this.btnPublicCreateCozyFolder,
        'Create my Cozy Button'
      )
```

When you create a if / else block we're excepting that you do very distinct stuff. This is not the case here, you just change a selector so an inline if is more readable?